test/tap/groups/ai/setup-infras.bash

```
@@ -0,0 +1,133 @@
+# Validate that required infrastructure names were resolved
+if [ -z "${DEFAULT_MYSQL_INFRA:-}" ]; then
+    echo "ERROR: Could not resolve DEFAULT_MYSQL_INFRA (check ${SCRIPT_DIR}/infras.lst)"
+    exit 1
+fi
+
+if [ -z "${DEFAULT_PGSQL_INFRA:-}" ]; then
+    echo "ERROR: Could not resolve DEFAULT_PGSQL_INFRA (check ${SCRIPT_DIR}/infras.lst)"
+    exit 1
+fi
+
+export ROOT_PASSWORD=$(echo -n "${INFRA_ID}" | sha256sum | head -c 10)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)